### DISCUSSION TITLE
New release available

### DISCUSSION THREAD

**Discussion by nbaars** [WebGoat/WebGoat#1384] [Announcements]
With great pleasure, we present you with a new release of WebGoat 2023.3. Finally, it has been a while. This year starts with a new release of WebGoat.

For more details see: https://github.com/WebGoat/WebGoat/releases/tag/v2023.3

Kind regards,
Team WebGoat.

**Comment by zubcevic** [comment]
Nice, good start. Let's start a new year with a new release and some ideas to further improve and extend WebGoat

**Reply by zubcevic** [reply]
Let's also remove the version 8 from the README
